### PR TITLE
#UM-13 tag filter

### PR DESCRIPTION
**Pull request by Havana82**
## Description

This pull request adds a color change and check mark functionality to the tag button.
A user can only select one tag at a time
## Acceptance Criteria
- [x] Filter options are displayed below the search bar.
- [x] Filter tags color change when clicked
- [ ] When I select one or more filters, the displayed profiles are narrowed down to match the selected criteria.
- [ ] Clearing the filters or selecting a new filter updates the search results accordingly.
- [ ] The number of results displayed adjusts based on the applied filters.

## What type of PR is this? (check all applicable)

|     | Type                       |
| --- | -------------------------- |
|   x  | :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing steps
Switch to this branch with the following command: git switch feature/UM-13-tag-filter
Install dependencies with the following command: npm i
Run the following scripts: npm run dev to start Vite's dev build & npm run tailwind to start the Tailwind CSS build
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and click on each tag to see added feature

## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |

### CODE REVIEW

File: src/assets/bioData.js
Comment (Havana82) [inline src/assets/bioData.js:1]: added one tag key, i believe 2 tag keys appear due to merge issues

File: src/components/Button.jsx
Comment (Havana82) [inline src/components/Button.jsx:1]: created a separate component for the tag buttons for easy manipulation

File: src/components/Card.jsx
Comment (Havana82) [inline src/components/Card.jsx:1]: use of use state and click event to change color of button once clicked

**Review by shyusu4 — APPROVED** [review]
Looks good, are you going to add the filtering functionality in the next PR?

**Comment by Havana82** [comment]
> Looks good, are you going to add the filtering functionality in the next PR?

Yes! will do that by the end of today

**Pull request closed without merging**